letsencrypt/client/configuration.py

```diff
@@ -48,6 +48,18 @@ def cert_key_backup(self):  # pylint: disable=missing-docstring
             self.namespace.work_dir, constants.CERT_KEY_BACKUP_DIR,
             self.namespace.server.partition(":")[0])
 
+    @property
+    def accounts_dir(self):  #pylint: disable=missing-docstring
+        return os.path.join(
+            self.namespace.config_dir, constants.ACCOUNTS_DIR,
+            self.namespace.server.partition(":")[0])
```

Comment:
Hmm... I thought it would cause more trouble having multiple accounts under the same domain (not knowing which recovery token is actually needed/required).  Having multiple accounts that will conflict when minor changes are made.  I would tend to believe that the server would share state on the domain.  That being said... it is probably safer overall to distinguish between any servername changes.

It is unlikely to be an issue, and when it is, it will just cause confusion instead of being a complete error.